workers/unity/Assets/Playground/Config/CubeTemplate.cs

```diff
@@ -12,30 +13,23 @@ public static Entity CreateCubeEntityTemplate(Coordinates coords)
         {
             const string entityType = "Cube";
 
-            var transform = TransformInternal.Component.CreateSchemaComponentData(
-                new Location((float) coords.X, (float) coords.Y, (float) coords.Z),
-                new Quaternion(1.0f, 0.0f, 0.0f, 0.0f),
-                new Velocity(0.0f, 0.0f, 0.0f),
-                0,
-                0.0f
-            );
-
             var cubeColor = CubeColor.Component.CreateSchemaComponentData();
             var cubeTargetVelocity = CubeTargetVelocity.Component.CreateSchemaComponentData(new Vector3f { X = -2.0f });
             var launchable = Launchable.Component.CreateSchemaComponentData(new EntityId(0));
 
-            var entity = EntityBuilder.Begin()
+            var entityBuilder = EntityBuilder.Begin()
                 .AddPosition(coords.X, coords.Y, coords.Z, WorkerUtils.UnityGameLogic)
                 .AddMetadata(entityType, WorkerUtils.UnityGameLogic)
                 .SetPersistence(true)
                 .SetReadAcl(WorkerUtils.AllWorkerAttributes)
-                .AddComponent(transform, WorkerUtils.UnityGameLogic)
                 .AddComponent(cubeColor, WorkerUtils.UnityGameLogic)
                 .AddComponent(cubeTargetVelocity, WorkerUtils.UnityGameLogic)
-                .AddComponent(launchable, WorkerUtils.UnityGameLogic)
-                .Build();
+                .AddComponent(launchable, WorkerUtils.UnityGameLogic);
+
+            TransformSynchronizationHelper.AddComponents(ref entityBuilder, WorkerUtils.UnityGameLogic,
```

Comment:
Is there a way to do with without breaking the fluent API?

Reply:
/whispers "extension methods"

Reply:
It was a leading question 😉